page_templates/developer_guide_how_to.rst

```diff
@@ -0,0 +1,69 @@
+How To Template (Use Title Case)
+============================================================
+
+Section headers use sentence case
+
```

Comment:
maybe we can add `meta` field here for seo purpose, and table of contents.

Reply:
Hmm yes good point, but should we have a predefined list of keywords to choose from? or people create their own?

Reply:
I'm not so sure about table of contents. Ideally, the page should be short enough not to have one :)

Reply:
You mean adding this? https://docs.readthedocs.io/en/stable/guides/technical-docs-seo-guide.html#use-meta-tags

Reply:
yes, we can also add `keywords` to `meta`

Reply:
I think these are good ideas, but maybe it is too much for the moment? I don't want to have developers do to much work